lib/bike_brigade_web/live/campaign_live/form_component.ex

```diff
@@ -72,7 +72,7 @@ defmodule BikeBrigadeWeb.CampaignLive.FormComponent do
 
   @impl true
   def mount(socket) do
-    programs = for p <- Delivery.list_programs(), do: {p.name, p.id}
+    programs = for p <- Delivery.list_programs(), do: [key: p.name, value: p.id, label: p.name]
```

Comment:
This was required in order to add "label" attributes to options in the `<select>` tag; these labels are used to traget dropdowns in tests.